pkg/util/stats_tracker.go

```diff
@@ -161,3 +162,21 @@ func (s *StatsTracker) dropOldPoints(now int64) {
 		s.peakPointsHead = nil
 	}
 }
+
+func (s *StatsTracker) InfoKey() string {
+	return "Latency Stats"
+}
+
+func (s *StatsTracker) Info() []string {
+	AllTimeAvgLatency := s.AllTimeAvg() / int64(time.Millisecond)
+	AllTimePeakLatency := s.AllTimePeak() / int64(time.Millisecond)
+	RecentAvgLatency := s.MovingAvg() / int64(time.Millisecond)
+	RecentPeakLatency := s.MovingPeak() / int64(time.Millisecond)
+
+	return []string{
+		"Average Latency (ms): " + strconv.FormatInt(RecentAvgLatency, 10),
```

Comment:
NIT: I'd prefer using `SPrintF` over `FormatInt`
```suggestion
		fmt.Sprintf("Average Latency (ms): %d", RecentAvgLatency)
```
(as well as for the others below)